src/commands/Scheduled.ts

```diff
@@ -0,0 +1,116 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction } from 'discord.js';
+import Command from '../Command';
+import { BotClient } from '../types';
+import { DateTime } from 'luxon';
+import schedule from 'node-schedule';
+import { v4 as newUUID } from 'uuid';
+import Logger from '../utils/Logger';
+
+/**
+ * This command will send a scheduled message after waiting
+ * for a user specified time.
+ * 
+ * 
+ * The command will run a cronjob to send the message. Currently
+ * there is a 1 day limit on scheduled messages but that will likely
+ * change soon.
+ */
+export default class Scheduled extends Command {
+  constructor(client: BotClient) {
+    const definition = new SlashCommandBuilder()
+      .setName('schedulesend')
+      .addStringOption((option) => option.setName('message')
+        .setDescription('message to be sent')
+        .setRequired(true))
+      .addStringOption((option) => option.setName('wait')
+        .setDescription('Time to wait before sending message. Use hh:mm format!')
+        .setRequired(true))
+      .setDescription('sends a scheduled message to the channel');
+
+ 
+    super(client, {
+      name: 'schedulesend',
+      boardRequired: true,
+      enabled: true,
+      description: 'sends a scheduled message',
+      category: 'Utility',
+      usage: client.settings.prefix.concat('schedulesend'),
+      requiredPermissions: ['SEND_MESSAGES'],
+    }, definition);
+  }
+  
+  public async run(interaction: CommandInteraction): Promise<void> {
+
+    /**
+     * User input values
+     */
+    const message = interaction.options.getString('message', true);
+    const member = interaction.member;
+    const wait = interaction.options.getString('wait', true);
+
+    /**
+     * Arrays to help check whether time values are valid
+     * and to configure Date object
+     */
+    let timeArray = wait.split(':'); 
+
+    //Checks if the timestring is valid
+    //If there exists a timestring
+    if (wait) {
+      let time = DateTime.fromFormat(wait, 'hh:mm');
+      if (!time.isValid){
+        super.respond(interaction, 
+          { content: 'Invalid wait! Use hh:mm formatting (e.g. 04:04)'
+            , ephemeral: true });
+        return; 
+      }
+    }
+
+    //Gets the current time
+    let date = DateTime.now();
+
+    //Creates date object with time to send
+    let dateToSend = date.plus({ hours: Number(timeArray[0]), minutes: Number(timeArray[1]), seconds: 3 });
+    
+    const messageReceived = `Message Received! I will send it at <t:${Math.trunc(dateToSend.toSeconds())}:F>`;
+    //If it passes all above edge cases then reply...
+    await super.respond(interaction, {
+      content: messageReceived,
+      ephemeral: false,
+    });
+
+  
+    //Message to be sent
+    const messageToSend = `**Scheduled Message from ${member}:**` + `\n\n ${message}`;
```

Comment:
Also a nitpick but there's no need to concatenate template literal strings. It can be written in a single literal like `${member}:**\n\n ${message}`